ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/Uniform.hs

```diff
@@ -134,3 +146,47 @@ prop_uniformSchedule = do
     schedulerConfig = (noTimeoutsSchedulerConfig scheduleConfig) {scEnableGdd = True}
 
     scheduleConfig = defaultPointScheduleConfig
+
+-- | Test that the leashing attacks do not delay the immutable tip
+--
+-- This test is expected to fail because we don't test a genesis implementation
+-- yet.
+prop_leashingAttackNoProgress :: QC.Gen QC.Property
+prop_leashingAttackNoProgress = QC.expectFailure <$> do
```

Comment:
`QC.expectFailure` causes the trace of the first failing test to be printed to stdout. The trace of running the scheduler is very long. I'd like to make it shorter, but I don't know what flag to use for that. Please, let me know if you have any advise.